lib/active_admin_versioning/active_admin/dsl.rb

```diff
@@ -43,7 +43,13 @@ def versioning
       private
 
       def enabled_paper_trail?
-        config.resource_class.try(:paper_trail).try(:enabled?) || config.resource_class.try(:paper_trail_enabled_for_model?)
+        is_enabled = false
+        if config.resource_class.try(:paper_trail)
+          is_enabled = config.resource_class.try(:paper_trail).try(:enabled?)
+        else
+          is_enabled = config.resource_class.try(:paper_trail_enabled_for_model?)
+        end
+        return is_enabled
```

Comment:
Thanks for sending pull request.
But I suppose better that it return true or falsey (nil or false) without setting local variable.

```ruby
def enabled_paper_trail?
  if config.resource_class.respond_to?(:paper_trail)
    config.resource_class.paper_trail.try(:enabled?)
  else
    config.resource_class.try(:paper_trail_enabled_for_model?)
  end
end
```